### ISSUE TITLE
Broken test: PrintInfoIT

### ISSUE THREAD

**Issue by DomGarguilo** [apache/accumulo#4649]
**Test name(s)**
- PrintInfoIT.testRFileContainingSummaryInfo
- PrintInfoIT.testRFileNotContainingSummaryInfo

**Describe the failure observed**
The mentioned test cases are failing in `main` and `elasticity`. They pass for me in `2.1`.
```
[INFO] Running org.apache.accumulo.test.PrintInfoIT
[ERROR] Tests run: 3, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 17.08 s <<< FAILURE! -- in org.apache.accumulo.test.PrintInfoIT
[ERROR] org.apache.accumulo.test.PrintInfoIT.testRFileNotContainingSummaryInfo -- Time elapsed: 3.946 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Call to PrintInfo failed ==> expected: <0> but was: <1>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:563)
	at org.apache.accumulo.test.PrintInfoIT.execPrintInfo(PrintInfoIT.java:197)
	at org.apache.accumulo.test.PrintInfoIT.testRFileNotContainingSummaryInfo(PrintInfoIT.java:91)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)

[ERROR] org.apache.accumulo.test.PrintInfoIT.testRFileContainingSummaryInfo -- Time elapsed: 2.571 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Call to PrintInfo failed ==> expected: <0> but was: <1>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:563)
	at org.apache.accumulo.test.PrintInfoIT.execPrintInfo(PrintInfoIT.java:197)
	at org.apache.accumulo.test.PrintInfoIT.testRFileContainingSummaryInfo(PrintInfoIT.java:113)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
```

**Testing Environment:**
 - Version of this project: main, elasticity
 - current commit: 69584e2835

**Comment by kevinrr888** [comment]
I haven't looked into this but https://github.com/apache/accumulo/pull/4634/commits/0c1c774be47754a765a515911a91af18bce265bf may be the commit related to this

**Comment by DomGarguilo** [comment]
Fixed via #4658
